Makefile

```diff
@@ -83,9 +84,25 @@ release:				#: Build the release version
 	$(MAKE) build BUILD_DIR=release
 
 min_debug:				#: Minimal build with debugging symbols
-	$(MAKE) cmake BUILD_DIR=debug BUILD_TYPE=debug EXTRA_CMAKE_FLAGS=-DVELOX_BUILD_MINIMAL=ON
+	$(MAKE) cmake BUILD_DIR=debug BUILD_TYPE=debug EXTRA_CMAKE_FLAGS="-DVELOX_BUILD_MINIMAL=ON -DVELOX_ENABLE_BENCHMARKS=ON"
 	$(MAKE) build BUILD_DIR=debug
 
+benchmarks-build:
+	$(MAKE) cmake BUILD_DIR=release BUILD_TYPE=Release \
+			EXTRA_CMAKE_FLAGS="-DVELOX_BUILD_MINIMAL=ON -DVELOX_ENABLE_BENCHMARKS=ON"
+	$(MAKE) build BUILD_DIR=release
+
+benchmarks-baseline:
+	$(MAKE) benchmarks-build
+	mkdir -p $(BENCHMARK_DIR)/dumps
+	$(BENCHMARK_DIR)/velox_benchmark_filter_benchmark --json \
```

Comment:
Curious - Did you add support for that json flag or does folly benchmark already support it ?